index.bs

```diff
@@ -1202,6 +1242,9 @@ requested credential exactly matches the RP ID supplied by the client.
 The `TUP` flag SHALL be set if and only if the authenticator detected a user through an authenticator specific gesture. The
 `RFU` bits in the flags byte SHALL be set to zero.
 
+The `RK` flag SHALL be set if and only if the authenticator is capable of storing <a>credential private key</a> to 
+    generate the [RP]-specific <a>credential key pair</a> on demand. 
```

Comment:
I'm having difficulty parsing this. Again, why not focus on this specific credential and say that it can be looked up and used in the future by this authenticator based only on RP ID?